modules/olm-accessing-images-private-registries.adoc

```diff
@@ -55,7 +55,19 @@ A registry credentials file can, by default, store details for more than one reg
                         "auth": "FrNHNydQXdzclNqdg=="
                 },
                 "quay.io": {
-                        "auth": "Xd2lhdsbnRib21iMQ=="
+                        "auth": "fegdsRib21iMQ=="
+                }
+                },
+                "https://quay.io/my-namespace/my-user/my-image": {
+                        "auth": "eWfjwsDdfsa221=="
+                }
+                },
```

Comment:
Annoyingly, the comma here and in line 68 render in red highlighting in the preview build, but no idea why. 🤷

Reply:
I'm not sure either. Can't make the highlight go away. Even copied/pasted the previous brackets and comma.